kolibri/plugins/audio_mp3_render/kolibri_plugin.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""
+Copy and modify this code for your own plugin.
+"""
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+
+from kolibri.plugins.base import KolibriFrontEndPluginBase
+from kolibri.plugins.hooks import CONTENT_RENDERER_ASYNC, FRONTEND_PLUGINS
+
+logger = logging.getLogger(__name__)
+
+
+class KolibriAudioMP3RenderFrontEnd(KolibriFrontEndPluginBase):
+    """
+    The base learn code for the learn page.
+    """
+    entry_file = "assets/src/render_audio_mp3_module.js"
+
+    events = {
+        "content_render:audio/mp3": "render"
```

Comment:
Ok, good to demo event registration. Have you considered appropriate behavior if multiple plugins register to this callback? Is there a use case for that?